README.md

```diff
@@ -9,26 +9,25 @@
 [![Netlify Status](https://api.netlify.com/api/v1/badges/4fa2c17a-6385-4cc6-9919-e32c134175d9/deploy-status)](https://app.netlify.com/sites/blazorstatic/deploys)
 
 
-Embrace the capabilities of Blazor on .NET 8 to craft static websites.
+Embrace the capabilities of [Blazor](https://dotnet.microsoft.com/en-us/apps/aspnet/web-apps/blazor) on .NET to craft static websites.
 
-Transform your Blazor app into a static site generator.
+Transform your Blazor app into a static site.
 
 # 🔧 How does it work?
 
-Like any .NET Blazor app! That's the main point. If you know how to build a Blazor app, you can transfer it to HTML files easily.
-**BlazorStatic** will help you with that. It can also help with processing markdown files.
+**BlazorStatic** aims to be intuitive and easy to use for most developers who have worked with any kind of front-end framework, especially if you already work with Blazor.
 
-> Website for BlazorStatic is a feature-rich example of the library. Check the [source code](https://github.com/tesar-tech/BlazorStatic/tree/master/BlazorStaticWebsite). 
+> The site for BlazorStatic is a feature-rich example of the library. Check the [source code](https://github.com/tesar-tech/BlazorStatic/tree/master/BlazorStaticWebsite). 
 
-Upon running your app, **BlazorStatic** fetches the pages' HTML with `HttpClient`, outputs HTML files and assets to the `output` folder, which you can deploy to any static hosting service.
+Upon running your app, **BlazorStatic** fetches the pages' HTML with `HttpClient`, outputs HTML files and assets to the `output` folder, and from there you can deploy to any standard HTTP server or static site hosting service.
```

Comment:
what about: "to any server or hosting"?

Reply:
I think it's nice to make it clear that it's just a bundle of files that an HTTP server could host, but I don't really have a strong opinion either way.

Reply:
for me it's hard to figure out what "standart HTTP server" means. It is probably technically correct, but doesn't imply the solution is easy..

```md
Upon running your app, **BlazorStatic** fetches the pages' HTML with `HttpClient`, outputs HTML files and assets to the `output` folder, ready to be [deployed almost anywhere](https://tesar-tech.github.io/BlazorStatic/docs/deployment), often for free.
```

Reply:
Any HTTP server (nginx, caddy, Apache) can host your files. You are not limited to a hosting service, which is part of why SSG is so popular: it's simple.